lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -395,6 +441,7 @@ def filter(tag, time, record)
           log.error("output #{time} #{tag} #{record}")
         end
       end
+      transform_event(tag, record)
```

Comment:
This should be much higher in the code, just before `add_pipeline_metadata`, unless you rewrite this to work as a `formatter`

Reply:
I have already commented on your recommendation here https://github.com/ViaQ/fluent-plugin-viaq_data_model/pull/9#discussion_r137520308

In my testing, if I did what you recommended, neither of these two transformations were performed due to reasons explained there.
```
pipeline_metadata.collector.original_raw_message <- message
@dest_time_name                                  <- event.metadata.creationTimestamp
```
Could you please advise what solution do you recommend to my concerns?

Reply:
Sorry, I missed that earlier comment.  I suggest changing the `add_pipeline_metadata` function to add whatever you need.